src/neptune/new/internal/disk_queue.py

```diff
@@ -233,4 +233,5 @@ def __enter__(self):
         return self
 
     def __exit__(self, exc_type, exc_val, exc_tb):
```

Comment:
We're not using queue as context manager anymore, but if you want we can leave it.